README.md

```diff
@@ -744,6 +746,10 @@ object ImplicitHolder {
 }
 ```
 
+### <a name='symbol'>Symbol Literals</a>
+
+__Avoid using symbol literals__. Symbol literals (e.g. `'column`) were deprecated as of Scala 2.13 by [Proposal to deprecate and remove symbol literals](https://contributors.scala-lang.org/t/proposal-to-deprecate-and-remove-symbol-literals/2953). Apache Spark used to leverage this syntax to provide DSL; however, now it started to remove this deprecated usage away. See also [SPARK-29392](https://issues.apache.org/jira/browse/SPARK-29392).
```

Comment:
Yep this is fine. You can add that you should use `$"col"` instead to provide a Column or just `"col"` to provide a column name instead, but this may be obvious.

Reply:
I just wanted to make it sounds like a more general Scala guide :)..

Reply:
OK sounds good